catroid/src/main/java/org/catrobat/catroid/content/actions/RepeatAction.java

```diff
@@ -67,7 +67,7 @@ public boolean delegate(float delta) {
 		if (executedCount >= repeatCountValue && !isForeverRepeat) {
 			return true;
 		}
-		if (action.act(delta) && currentTime >= LOOP_DELAY) {
+		if (action != null && action.act(delta) && currentTime >= LOOP_DELAY) {
```

Comment:
The action is recreated by handleStopAllScripts() in the Look class.
However as @cheidenreich suggested I'll put that logic directly into the StopAllScriptsAction.